Sources/Sentry/Public/SentryTraceContext.h

```diff
@@ -75,6 +80,20 @@ NS_SWIFT_NAME(TraceContext)
                         sampled:(nullable NSString *)sampled
                        replayId:(nullable NSString *)replayId;
 
+/**
+ * Initializes a SentryTraceContext with given properties.
+ */
+- (instancetype)initWithTraceId:(SentryId *)traceId
```

Comment:
Same as in https://github.com/getsentry/sentry-cocoa/pull/4751#discussion_r1930846919